src/layer/eth/neighbor.rs

```diff
@@ -126,11 +133,13 @@ impl<'a> Cache<'a> {
     pub fn fill(
         &mut self,
         protocol_addr: IpAddress,
-        hardware_addr: EthernetAddress,
+        hardware_addr: Mapping,
```

Comment:
Ich hätte `fill` ja nicht verändert, sondern eine neue Methode eingeführt um ein `LookingFor` einzutragen. Zumindest nach außen hin das schönere Interface.

Reply:
Und es spart eine Menge unnötiger Modifikationen in `ip` und `eth`, die sich ja eben nicht mit `arp` befassen wollen.